src/timeline/index.html

```diff
@@ -75,13 +75,14 @@
 		<div tl-scrollable-tracks id="scrolling_tracks">
 			<div id="track-container" tl-track tl-multi-selectable style="width: {{getTimelineWidth(1024)}}px; padding-bottom: 2px;">
 				<!-- TRACKS -->
-				<div ng-repeat="layer in project.layers.slice().reverse()" id="track_{{layer.number}}" ng-right-click="showTimelineMenu($event, layer.number)"  class="{{getTrackStyle(layer.lock)}}" style="width:{{getTimelineWidth(1024)}}px;">
+				<div ng-repeat="layer in project.layers.slice().reverse()" id="track_{{layer.number}}" ng-right-click="showTimelineMenu($event, layer.number)" class="{{getTrackStyle(layer.lock)}}">
```

Comment:
Another unexpected optimization: I noticed we were forcing the `width` of every track `DIV` with inline styling. But they're all in a container that's _also_ forced with **the exact same width**. So I just added `width: 100%` to the track CSS, removed the inline styling, and let the container dictate the width of the tracks.